src/components/UI/CodeBlock/CodeBlock.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { vars } from "../../../styles";
+import { SvgIcon, SvgIconCopy } from "../SvgIcon";
+
+const CodeBlockWrap = styled.div`
+  background-color: ${vars.blueLighter};
+  color: ${vars.greyDark};
+  padding: 16px;
+  height: 160px;
+  position: relative;
+  overflow-y: scroll;
+  border-radius: ${vars.buttonRadius};
+
+  svg {
+    color: ${vars.blue};
+    cursor: pointer;
+
+    &:hover {
+      color: ${vars.blueDark};
+    }
+  }
+
+  .copy-icon {
+    bottom: 16px;
+    right: 16px;
+  }
+`;
+
+interface CodeBlockProps {
+  code: string;
+}
+
+export const CodeBlock: React.FunctionComponent<CodeBlockProps> = ({ code }: CodeBlockProps) => {
+  const copyToClipboard = (): void => {
+    const str = document.getElementById("copy-textarea")?.innerText || "";
+    // Needs to be input element to copy so created a temp
+    const $temp = document.createElement("textarea");
```

Comment:
why $temp ah? seems weird to use $ we don't do that anywhere else
just call it temporaryDomElement or something